src/object/elo.c

```diff
@@ -288,6 +318,11 @@ elo_copy (DB_ELO * elo, DB_ELO * dest)
 		ret = er_errid ();
 		goto error_return;
 	      }
+#ifdef SERVER_MODE
+	    mmon_add_stat_with_tracking_tag (strlen (elo->locator) + 1);
+	    strcpy (out_uri, elo->locator);
```

Comment:
why is this string copy needed?

Reply:
Because if the function goes to fail, it has to free allocated memories, but we don't know which case is hit in this circumstance. So I have to make recognize all string allocations at once. `out_uri` is a static char array.

Reply:
If the length of the string is required when the it fails, then we don't need to copy the string I think. Instead we can store the length somewhere

Reply:
I checked it and I agree that we don't need to use `out_uri` and strcpy. Done.